### PR TITLE
ceph & cinder integration

### PR DESCRIPTION
**Pull request by msambol**
Don't need all eyes on this yet, just the Ceph team.

### CODE REVIEW

File: envs/example/defaults.yml [hunk 1]
```diff
@@ -107,8 +107,11 @@ nova:
     debug: True
     verbose: True
 
+ceph:
+  enabled: false
+
 cinder:
-  enabled: True
+  enabled: {{ ceph.enabled }}
```
Comment (cjchang) [inline envs/example/defaults.yml]: Just curious, can you enable cinder without enabling ceph?

Comment (msambol) [inline envs/example/defaults.yml]: A question I have myself. I think the direction we are going is if the customer doesn't get ceph, they don't get cinder. But need to hear back from Vinit or Niraj.

**Comment by msambol** [comment]
I've moved this here: https://github.com/blueboxgroup/ursula/pull/1140

File: envs/example/defaults.yml (hunk 1 above)
Comment (cjchang) [inline envs/example/defaults.yml]: I assume that the cinder is supported today without ceph. If the change for cinder to work only with ceph, there may be some issues with the current deployments unless none enabled teh cinder...

Comment (msambol) [inline envs/example/defaults.yml]: Agreed. Take a look at discussion here: https://github.com/blueboxgroup/ursula/pull/1140.

**Pull request merged**